### PR TITLE
CBBD-137 handle nullable fields

### PR DESCRIPTION
**Pull request by dho937**

### CODE REVIEW

**Comment by dho937** [comment]
@karlmdavis @sbrockhoff  Please review when you get a chance.  Thanks.

**Comment by dho937** [comment]
@karlmdavis @sbrockhoff I need to merge this soon.  Can anyone do a quick review?  Thanks.

File: bluebutton-data-pipeline-fhir-load/src/main/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformer.java [hunk 1]
```diff
@@ -724,16 +724,19 @@ private TransformedBundle transformCarrierClaim(RifRecordEvent<CarrierClaimGroup
...
-		ReferralRequest referral = new ReferralRequest();
-		referral.setStatus(ReferralStatus.COMPLETED);
-		referral.setPatient(referencePatient(claimGroup.beneficiaryId));
-		referral.addRecipient(referrerReference);
-		// Set the ReferralRequest as a contained resource in the EOB:
-		eob.setReferral(new Reference(referral));
+		if (claimGroup.referringPhysicianNpi.isPresent()) {
+			Practitioner referrer = new Practitioner();
+			referrer.addIdentifier().setSystem(CODING_SYSTEM_NPI_US)
+					.setValue(claimGroup.referringPhysicianNpi.toString());
```
Comment (karlmdavis) [inline bluebutton-data-pipeline-fhir-load/src/main/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformer.java]: Isn't using `toString()` here (and, it looks like, on every other `Optional` a fairly major bug? I'm looking at the code for it, and it looks like it should be returning something like the following: `String.format("Optional[%s]", value)`.

If I'm right about that being a bug, I think this also illustrates a problem with your tests, that they didn't catch it. You should add at least a couple sanity checks in there that would verify this.

Even if I'm wrong and this just returns the bare value, it's more semantically correct to use `Optional.get()` on all of these.

File: bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/HHAClaimGroup.java [hunk 2]
```diff
@@ -1,1173 +1,1176 @@
-package gov.hhs.cms.bluebutton.datapipeline.rif.model;
```
Comment (karlmdavis) [inline bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/HHAClaimGroup.java]: Looks like our line-ending fix has us in for some pain until we go through and fix every file that was committed with `CRLF` line endings. (Git is now converting those on commit to Unix line endings.)

Can you please file a bug for me in Sprint 11 to go through and do all of these at once? Diffs will be useless on many files for a while unless/until we do.

**Pull request merged**